src/util/measurementUncertaintyTool.js

```diff
@@ -0,0 +1,66 @@
+import Decimal from 'decimal.js';
+
+// Run node version with 14.18.3
+/**
+ * Custom measurement uncertainty rounding rules.
+ * @param  {number} inputValue            measured value from annotation
+ * @param  {number} uncertaintyValue      uncertainty from pixel diagonal
+ */
+
+/**
+ * Rounding
+ */
+
+function roundValue(inputValue, uncertaintyValue) {
+  const value = new Decimal(inputValue);
+  let i = 0;
```

Comment:
i is very generic for a variable name and lacks the context to immediately associate it to a for loop int, instead, we can use for(let i; ...)